data.sql

```diff
@@ -70,4 +70,11 @@ INSERT INTO visits (animal_id, vet_id, visit_date) VALUES ((SELECT id FROM anima
 INSERT INTO visits (animal_id, vet_id, visit_date) VALUES ((SELECT id FROM animals WHERE name = 'Boarmon'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), 'February 27, 2020');
 INSERT INTO visits (animal_id, vet_id, visit_date) VALUES ((SELECT id FROM animals WHERE name = 'Boarmon'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), 'August 3, 2020');
 INSERT INTO visits (animal_id, vet_id, visit_date) VALUES ((SELECT id FROM animals WHERE name = 'Blossom'), (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), 'May 24, 2020');
-INSERT INTO visits (animal_id, vet_id, visit_date) VALUES ((SELECT id FROM animals WHERE name = 'Blossom'), (SELECT id FROM vets WHERE name = 'William Tatcher'), 'January 11, 2021');
\ No newline at end of file
+INSERT INTO visits (animal_id, vet_id, visit_date) VALUES ((SELECT id FROM animals WHERE name = 'Blossom'), (SELECT id FROM vets WHERE name = 'William Tatcher'), 'January 11, 2021');
+
+-- SECOND WEEK (Pair-programming)
+
+explain analyze SELECT COUNT(*) FROM visits where animal_id = 4;
+explain analyze SELECT * FROM visits where vet_id = 2;
```

Comment:
- [ ] Great job on analyzing these queries well and taking actions to reduce their execution time. However, this query
 ```sql
      SELECT * FROM visits where vet_id = 2;
 ```
 Does not show any reduction in execution time despite the actions you have taken, rather it has increased according to my observation (it is greater than 1000ms) while it is supposed to be smaller than 1000ms (I am talking about the execution time) 😃 
![image](https://user-images.githubusercontent.com/90222110/191565213-1373e3ce-d15d-41c7-80e7-8b2189e2b403.png)
So,  please go through the process again and try to reduce its execution time. You can use different techniques learned like adding index and others 😉 🏃

Reply:
Hi, @BertrandConxy. That one actually works, the thing is that we added to many new data into the table since it was running so fast so we had to repeat step 3 some more times... It was using 4.000ms and after creating the Indexes it went as it shows in the picture, near 2.000ms 💹